Userland/Libraries/LibWeb/HTML/HTMLAnchorElement.cpp

```diff
@@ -66,6 +67,12 @@ Optional<String> HTMLAnchorElement::hyperlink_element_utils_referrerpolicy() con
     return attribute(HTML::AttributeNames::referrerpolicy);
 }
 
+Vector<StringView> HTMLAnchorElement::hyperlink_element_utils_subject_link_types() const
+{
+    String rel = MUST(this->rel().to_lowercase());
+    return rel.bytes_as_string_view().split_view_if(Infra::is_ascii_whitespace);
+}
```

Comment:
You're right. That is unsafe. Maybe structuring it like this would be better?
```cpp
bool HTMLAnchorElement::hyperlink_element_utils_includes_noreferrer_link_type() const
{
    String rel = MUST(this->rel().to_lowercase());
    return rel.bytes_as_string_view().split_view_if(Infra::is_ascii_whitespace).contains_slow("noreferrer"sv);
}
```